packages/rspeedy/create-rspeedy/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # create-rspeedy
 
+## 0.10.1
+
+### Patch Changes
+
+- Fix example missing color: `color: rgba(255, 255, 255, 0.65)`, because inheritance in Harmony is not support.
+
```

Comment:
The `CHANGELOG.md` is generated by the changeset bot. You don't need to manually update it.

```suggestion
```